### PR TITLE
Added 0.1em letter spacing to H1

### PR DESCRIPTION
**Pull request by dalynewspaper**
I felt the Bangers text for H1 was a little squashed so I added a little letter spacing

### CODE REVIEW

**Comment by amn41** [comment]
hey bud you added some extra changes unrelated to the spacing

**Comment by amn41** [comment]
@guainumbi should we plan to deprecate index.css entirely?

**Comment by guainumbi** [comment]
yes, eventually we can handle all styling from the global theme or the styled components directly. there should be no use for external stylesheets after that.

File: src/index.css [hunk 1]
```diff
@@ -35,7 +35,8 @@ button:focus, button:hover {
 
 h1 {
   font-family: "Bangers";
-  color: #222;
+  letter-spacing: 0.1em;
+  color: #202A25¸;
```
Comment (guainumbi) [inline src/index.css:39]: can be imported from the theme colors

**Comment by guainumbi** [comment]
changes have been included in the last styling PR (and Alan's commit) so we can close this PR. 
FYI - I added typography to our global theme and included the letter spacing.

**Comment by amn41** [comment]
awesome - thank you!

**Pull request closed without merging**